catroid/src/main/res/values-no/strings.xml

```diff
@@ -1753,7 +1753,7 @@ needs read and write access to it. You can always change permissions through you
     <string name="formula_editor_logic_comparison">sammenligningsoperatorer</string>
     <string name="formula_editor_object_general">generelle egenskaper</string>
     <string name="formula_editor_object_movement">bevegelse egenskaper</string>
-    <string name="formula_editor_device_visual_detection">Visual sensors</string>
+    <string name="formula_editor_device_face_detection">Visual sensors</string>
```

Comment:
undo this changes, nothing to do here, it will eventually be overwritten by the translations